strimzi/hatch.toml

```
@@ -0,0 +1,10 @@
+[env.collectors.datadog-checks]
+
+[[envs.default.matrix]]
+python = ["3.8"]
```

```suggestion
python = ["3.9"]
```
https://github.com/DataDog/integrations-core/pull/14701